package.json

```diff
@@ -18,45 +18,43 @@
   "license": "AGPL-3.0",
   "main": "dist/tify.js",
   "scripts": {
-    "build": "vue-cli-service build",
-    "e2e": "node tests/e2e/runner.js --headless",
-    "lint": "vue-cli-service lint",
-    "lint-fix": "vue-cli-service lint --fix",
+    "build": "vite build",
+    "dev": "vite",
+    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs --fix --ignore-path .gitignore",
+    "mock-api": "node tests/iiif-api/server.js",
     "postinstall": "node build/create-icons.js",
-    "postversion": "echo 'Update version in README.md, then run \"git add README.md && git commit --amend --no-edit && git tag v'$npm_package_version' && git push && git push --tags\", and finally \"npm publish\" to publish '$npm_package_name' on npmjs.com.'",
     "preversion": "npm install-clean && npm run test",
-    "serve": "vue-cli-service serve",
-    "serve-mock-api": "node tests/iiif-api/server.js",
-    "test": "npm run unit && npm run e2e",
-    "unit": "vue-cli-service test:unit",
+    "preview": "vite preview",
+    "test:unit": "vitest run --environment jsdom --root src/",
+    "test:e2e": "start-server-and-test preview http://localhost:4173 'cypress run --e2e'",
+    "test:e2e:dev": "start-server-and-test 'vite dev --port 4173' http://localhost:4173 'cypress open --e2e'",
     "version": "npm run build && git add dist"
   },
   "devDependencies": {
-    "@mdi/js": "^7.0.96",
-    "@vue/cli-plugin-babel": "^4.5.19",
-    "@vue/cli-plugin-e2e-cypress": "~4.5.19",
-    "@vue/cli-plugin-eslint": "^4.5.19",
-    "@vue/cli-plugin-unit-jest": "^4.5.19",
-    "@vue/cli-service": "^4.5.19",
-    "@vue/eslint-config-airbnb": "^6.0.0",
-    "@vue/test-utils": "^1.3.0",
-    "axios": "^0.27.2",
-    "babel-eslint": "^10.1.0",
-    "cross-spawn": "^7.0.3",
-    "eslint": "^7.32.0",
-    "eslint-plugin-cypress": "^2.12.1",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-vue": "^8.7.1",
-    "eslint-plugin-vuejs-accessibility": "^1.2.0",
-    "jest-canvas-mock": "^2.4.0",
-    "node-sass-glob-importer": "^5.3.2",
-    "openseadragon": "^3.0.0",
-    "sass": "^1.55.0",
-    "sass-loader": "^10.3.1",
+    "@mdi/js": "^7.2.96",
+    "@rushstack/eslint-patch": "^1.3.0",
+    "@vitejs/plugin-vue": "^4.2.3",
+    "@vue/eslint-config-airbnb": "^7.0.0",
+    "@vue/test-utils": "^2.3.2",
+    "canvas": "^2.11.2",
+    "click-outside-vue3": "^4.0.1",
+    "cypress": "^12.13.0",
+    "eslint": "^8.42.0",
+    "eslint-plugin-cypress": "^2.13.3",
+    "eslint-plugin-vue": "^9.14.1",
+    "jsdom": "^21.1.2",
+    "openseadragon": "3.0.0",
+    "sass": "^1.62.1",
```

Comment:
Do we still "sass" as dep?

Reply:
Yes. https://vitejs.dev/guide/features.html#css-pre-processors

Reply:
Ah ok, somehow the build still succeeded for me when I tried to remove "sass".